src/stream.jl

```diff
@@ -134,7 +134,9 @@ end
 function describe_log_streams(
     config::AWSConfig, log_group_name::AbstractString, params::AbstractDict
 )
-    return CloudWatch_Logs.describe_log_streams(log_group_name, params; aws_config=config)
+    full_params = Dict("logGroupName" => log_group_name)
+    append!(full_params, params)
```

Comment:
```suggestion
    full_params = Dict{String,Any}("logGroupName" => log_group_name)
    merge!(full_params, params)
```
Params passed in are not necessarily strings, so the new dict is given `Any` element type. `merge!` is like `append!` but for dicts.